sw360chores.pl

```diff
@@ -293,16 +296,17 @@ =head1 SYNOPSIS
         @args = @args?@args:();
 
         if(defined $ENV{'SW360CHORES_VERSION'}){
-            unshift @args, ("-t", "sw360/${name}:$ENV{'SW360CHORES_VERSION'}");
+            unshift @args, ("-t", "$projectName/${name}:$ENV{'SW360CHORES_VERSION'}");
```

Comment:
our image names should not depend on the project name. Since they should be usable in multiple deployments at the same time. What you should use would be tags.